behave/features/e2e_test_features/01.simple_web_submission_entry.feature

```diff
@@ -18,7 +18,7 @@ Feature: COVID-19 Shielded vulnerable people service - basic e2e user journey -
 
     Scenario: Should be re-directed to shielding because vulnerable when eligible postcode entered
```

Comment:
do we have any tests where shielding is false e.g for tier 4 or 3?

Reply:
All the current tier 3 tests are for shielding false.